cluster/images/hyperkube/Makefile

```diff
@@ -1,6 +1,9 @@
-# build the hyperkube image.
+# Build the hyperkube image.
+#
+# Usage:
+#   VERSION=v1.1.2 [REGISTRY="gcr.io/google_containers"] make build
 
-VERSION=v1.1.2
+REGISTRY?="gcr.io/google_containers"
```

Comment:
@2opremio has a cute trick for VERSION which uses:

    $(shell git rev-parse --short HEAD)

makes for a pretty nice image tag too.